docs/DocGenerator/lib/IndexData.py

```diff
@@ -0,0 +1,81 @@
+"""
+brief: Wazuh DocGenerator config parser.
+copyright: Copyright (C) 2015-2021, Wazuh Inc.
+date: August 04, 2021
+license: This program is free software; you can redistribute it
+         and/or modify it under the terms of the GNU General Public
+         License (version 2) as published by the FSF - Free Software Foundation.
+"""
+
+import os
+import re
+import json
+import requests
+from elasticsearch import Elasticsearch, helpers
+
+class IndexData:
+    """
+    brief: Class that indexes the data from JSON files into ElasticSearch.
+    """
+    def __init__(self, index, config):
+        self.conf = config
+        self.path = self.conf.documentation_path
```

Comment:
Is this assignment required? It will be more clear to directly use **self.conf.documentation_path**

Reply:
Done